parsers/jscript.c

```diff
@@ -1667,6 +1667,13 @@ static bool parseMethods (tokenInfo *const token, const tokenInfo *const class,
 				} while (! isType (token, TOKEN_EOF) && depth > 0);
 			}
 
+			if (is_dynamic_prop)
+			{
+				injectDynamicName (name, dprop);
+				dprop = NULL;
+			}
+			vStringDelete (dprop);
```

Comment:
the `vStringDelete (dprop)` could be in an else block (although it's safe as is as you set it to `NULL` in the if)

Reply:
Updated.